app/assets/javascripts/3dbio_viewer/src/webapp/components/dropdown/Dropdown.tsx

```diff
@@ -40,6 +41,7 @@ export function Dropdown<Id extends string = string>(
         leftIcon,
         expanded,
         deselectable,
+        disabled
```

Comment:
I see there is no trailing comma here, which should be automatically added by your editor (yarn prettify does it). Review your editor settings. Run yarn prettify to see other places formatting is also off.